### PR TITLE
[15.0][MIG] storage_backend_ftp

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

Depends on 
- #145

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and code review, LGTM

**Comment by LoisRForgeFlow** [comment]
@OCA/storage-maintainers this one is ready, could you review it?

File: storage_backend_ftp/components/ftp_adapter.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (simahawk) [inline storage_backend_ftp/components/ftp_adapter.py:2]: the license of the module is LGPL. Can you check all licenses in this module pls?

File: storage_backend_ftp/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (simahawk) [inline storage_backend_ftp/__manifest__.py]: the license of the module is LGPL. Can you check all license references in this module pls?

**Review by simahawk — CHANGES_REQUESTED** [review]
probably not related to the migration but there's a license problem

**Review by ivantodorovich — APPROVED** [review]

File: storage_backend_ftp/__manifest__.py (hunk 2 above)
Comment (JasminSForgeFlow) [reply storage_backend_ftp/__manifest__.py]: Hi,

Updated License details.

Thanks

Comment (simahawk) [reply storage_backend_ftp/__manifest__.py]: can you make a run on v14 too?

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@sbidoul Runboat fails like this
```
++ oca_list_external_dependencies deb
+ DEBIAN_FRONTEND=noninteractive
+ apt-get install -qq --no-install-recommends
+ DEBIAN_FRONTEND=noninteractive
+ apt-get -yqq install rsync
E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/r/rsync/rsync_3.1.3-8ubuntu0.1_amd64.deb  404  Not Found [IP: 91.189.88.152 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```
Have you seen it before?

**Comment by LoisRForgeFlow** [comment]
@simahawk good to merge?

**Comment by LoisRForgeFlow** [comment]
@simahawk The pre-commit failled with the [typical cache error](https://github.com/OCA/storage/runs/5598238952?check_suite_focus=true#step:4:129). I think you can just retry to merge.

**Comment by simahawk** [comment]
Can you run `copier -f update` to sync from the addon repo template and submit a PR?

**Comment by JasminSForgeFlow** [comment]
> 

Done, Thanks

**Comment by JasminSForgeFlow** [comment]
> Can you run `copier -f update` to sync from the addon repo template and submit a PR?

Done, Please check
Thanks

**Comment by simahawk** [comment]
@JasminSForgeFlow thanks but with " and submit a PR" I meant: don't do it in this PR :wink: 
At least don't squash that change on top of the migration commit :pray:

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow thanks but with " and submit a PR" I meant: don't do it in this PR wink At least don't squash that change on top of the migration commit pray

Sorry for the misunderstanding, I have updated PR.
Thanks

**Comment by JasminSForgeFlow** [comment]
Hi @simahawk,

Added PR for copier update
- https://github.com/OCA/storage/pull/177

**Pull request merged**